framework/test/TestingMacros.h

```diff
@@ -136,4 +136,23 @@ namespace cppmicroservices {
   STATEMENT ;                                                                                 \
   US_TEST_FOR_EXCEPTION_END(EXCEPTIONCLASS)
 
+
+#define US_TEST_FOR_NO_EXCEPTION_END()                                         \
+cppmicroservices::TestManager::GetInstance().TestPassed();                     \
+}                                                                              \
+catch (...) {                                                                  \
```

Comment:
I like the macro to test for no exceptions. I think it would be even more helpful to developers if any ```std::exception``` type information was displayed in the output.